docker/Dockerfile

```diff
@@ -6,7 +6,7 @@ MAINTAINER Hector Fernandez <hfernand@redhat.com>
 ENV LAUNCH_JBOSS_IN_BACKGROUND 1
 
 ENV KEYCLOAK_VERSION 3.0.0.Final
-ENV OPERATING_MODE clustered
```

Comment:
changing the ENV var value in the DC of OS would have been enough. but this also works